tailwind.config.js

```diff
@@ -62,6 +62,9 @@ module.exports = {
       width: {
         78: "19.375rem",
       },
+      maxWidth: {
```

Comment:
Don't you think that with changing the maxWidth you will change alot on the wrapper set I suggest you keep this and also add the maxwidth for the wrapper!